fares-data-build-tool/fdbt-netex-output
# feat: Include sales offer packages in NeTEx

# Description

-   This PR allows user defined sales offer packages to be included in the NeTEx they generate. This work replaces the placeholder sales offer package with dynamic user data received from the site.

# Testing instructions

-   Pull down this branch and run the `make` command in the `fdbt-dev` repo. In a separate shell, you should be able to run `make generate-validate-all` to trigger the netex convertor and validator and prove that the netex generator produces valid NeTEx.

# Type of change


# Checklist:
